components/molecules/Breadcrumb.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import BreadcrumbMenu from './BreadcrumbMenu';
+
+const baseClass = 'gds-page-header__breadcrumbs-list-item';
+const filterMenu = pathname => ({ path }) => path.charAt(0) !== ':' && !pathname.includes(path);
+
+const Breadcrumb = props => {
+    const {
+        config: { title, path, subpaths },
+        LinkElt,
+        pathname,
+        hideMenus,
+        isLast,
+        className = ''
+    } = props;
+    const hasSubpaths = !hideMenus && subpaths && subpaths.length;
+    const menu = hasSubpaths && subpaths.filter(filterMenu(pathname));
+    const hasMenu = !!(menu && menu.length);
+    const rootClass = cx(baseClass, className, {
+        [`${baseClass}--has-menu`]: hasMenu
+    });
+
+    return (
+        <li className={rootClass}>
+            {hasMenu && <BreadcrumbMenu LinkElt={LinkElt} menu={menu} path={path} />}
+            {isLast ? (
+                title
+            ) : (
+                <LinkElt className="gds-page-header__breadcrumbs-link" to={path}>
+                    {title}
+                </LinkElt>
+            )}
+        </li>
+    );
+};
+
+Breadcrumb.displayName = 'Breadcrumb';
+
+Breadcrumb.propTypes = {
+    LinkElt: PropTypes.oneOfType([PropTypes.element, PropTypes.func]).isRequired,
```

Comment:
Minor, but I don't like this prop name ;). I think I prefer `linkComponent` or even just `component`. Or other options?

Reply:
linkComponent sounds great, haha I didn't like it either but didn't have time to think about it.

Reply:
Fixed